vagrant/dev/bootstrap.sh

```diff
@@ -1,51 +1,25 @@
 #!/bin/sh
 
-# This script builds the base flocker-dev box.
+# This script performs the steps to build the base flocker-dev box until the
+# box must be rebooted.
 
 set -e
 
+yum update -y
 # Install useful yum repos
-yum install -y https://s3.amazonaws.com/archive.zfsonlinux.org/fedora/zfs-release$(rpm -E %dist).noarch.rpm
-curl https://copr.fedoraproject.org/coprs/tomprince/hybridlogic/repo/fedora-20-x86_64/tomprince-hybridlogic-fedora-20-x86_64.repo >/etc/yum.repos.d/hybridlogic.repo
+yum install -y https://s3.amazonaws.com/archive.zfsonlinux.org/epel/zfs-release$(rpm -E %dist).noarch.rpm
+yum install -y https://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-5.noarch.rpm
```

Comment:
`epel-release` should be avaiable by default on a fresh CentOS system, so no need for a URL.